site/docs/contribute.md

```diff
@@ -44,6 +44,7 @@ The Iceberg community prefers to receive contributions as [Github pull requests]
 * It's helpful to include a prefix in the summary that provides context to PR reviewers, such as `Build:`, `Docs:`, `Spark:`, `Flink:`, `Core:`, `API:`
 * If a PR is related to an issue, adding `Closes #1234` in the PR description will automatically close the issue and helps keep the project clean
 * If a PR is posted for visibility and isn't necessarily ready for review or merging, be sure to convert the PR to a draft
+* If a PR is already under review, please avoid force-pushing to the branch as it can make it difficult for reviewers to track changes. If you need to keep the PR up-to-date with the main branch, consider merge rather than rebase 
```

Comment:
> If you need to keep the PR up-to-date with the main branch, consider merge rather than rebase 

I have frequently used rebase and force push. from local branch, rebase is cleaner and easier to see the changes/commits in the PR source branch, as they are in the top.

trying to understand the difference btw merge and rebase in this case. can you share a PR where merge is used? want to see how it affect the look of the PR?

Reply:
I meant when I use `git log -n` on the source branch after merge. `merge` would get the new commits from main at the top.

Reply:
anyway, I am neural on this one. I generally don't see force push to be a problem. but agree that it is generally better not to amend earlier commits for the PR.